packages/firestore/lite/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@firebase/firestore/lite",
+  "name": "@firebase/firestore-lite",
```

Comment:
API Extractor considers this name invalid - looks like one slash is the maximum.

Reply:
Api-extractor doesn't support multiple entry points, so you need to trick it to think that lite is a package instead of a submodule. 

(I have a [PR](https://github.com/microsoft/rushstack/pull/1932) to support multiple entry points, but only for documentation and api report)

Reply:
The name actually doesn't matter, so it can be whatever that's easier for you. We use`/` to be consistent with how developer would use the submodules, so I prefer to stay this way. Does it give you any issue other than unpleasant validation error?

Reply:
I get the following error:

```
Error: Error parsing /Users/mrschmidt/GitHub/firebase/firebase-js-sdk/packages/firestore/lite/api-extractor.json:
The package name "@firebase/firestore/lite" contains an invalid character: "/"
```

The build stops after this message.

Reply:
oh right, api-extractor definitely doesn't like it. You can change it to `-` to work around it for now. I'm planning to release a forked version of api-extractor with support for multiple entry points. We will come back to this once that's done.